raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGrouper.java

```diff
@@ -44,17 +56,19 @@ public StreamsBuilder buildTopology(
       Map<String, Object> properties,
       StreamsBuilder streamsBuilder,
       Map<String, KStream<?, ?>> inputStreams) {
+
     Config jobConfig = getJobConfig(properties);
     String inputTopic = jobConfig.getString(INPUT_TOPIC_CONFIG_KEY);
     String outputTopic = jobConfig.getString(OUTPUT_TOPIC_CONFIG_KEY);
+    String bypassOutputTopic = jobConfig.getString(BYPASS_OUTPUT_TOPIC_CONFIG_KEY);
+    String outputTopicRawLogs = jobConfig.getString(OUTPUT_TOPIC_RAW_LOGS_CONFIG_KEY);
 
-    KStream<TraceIdentity, RawSpan> inputStream =
-        (KStream<TraceIdentity, RawSpan>) inputStreams.get(inputTopic);
+    KStream<byte[], JaegerSpanInternalModel.Span> inputStream =
```

Comment:
Needs to be an incremental change. We need to read from both `jaeger-spans` and `raw-spans-from-jaeger-spans` topics in this PR. 

Only in follow up PR some time down the line, we can just read from `jaeger-spans`